psp_notes.md

```diff
@@ -55,7 +55,7 @@ https://github.com/api3dao/airnode/blob/v1-protocol/packages/airnode-node/src/ev
 
 5. The Airnode makes the API call specified by the template and the additional parameters
 
-Probably current [call-api-ts](https://github.com/api3dao/airkeeper/blob/main/src/call-api.ts) in Airkeeper can be
+Probably current [call-api-ts](https://github.com/api3dao/airkeeper/blob/main/src/api/call-api.ts) in Airkeeper can be
```

Comment:
```suggestion
Probably current [call-api.ts](https://github.com/api3dao/airkeeper/blob/main/src/api/call-api.ts) in Airkeeper can be
```
Should be `call-api.ts`?

Reply:
You're right, it's weird that the tests are now passing.

Reply:
It can also be fixed from  #33  :smile:

Reply:
Sure, let's do that. I was going to leave it because they're psp_notes anyway, which will need to be processed into something suitable for public use.